frontend/src/utils/ropeApi.ts

```diff
@@ -34,6 +34,9 @@ export interface CourseBuild {
   courseShortName: string
   creatorEmail: string
   status: string
+  courseId?: string
+  courseEnrollmentUrl?: string
+  courseEnrollmentKey?: string
```

Comment:
Whoops, I think we currently don't actually populate these fields in the backend [here](https://github.com/openstax/rope/blob/0948b6b18cd82e0e4a336bb5879564c39565f261/backend/api/routers/moodle.py#L111). Right now if you manually set the values for a build in the DB they won't appear in the frontend. Can you make the change + perhaps add a test when fixing that little :bug: ?